apps/rush-lib/src/logic/Git.ts

```diff
@@ -133,11 +134,15 @@ export class Git {
   /**
    * Get the folder where Git hooks should go for the current working tree.
    * Returns undefined if the current path is not under a Git working tree.
+   * Respects `git config core.hooksPath`.
    */
   public getHooksFolder(): string | undefined {
     const repoInfo: gitInfo.GitRepoInfo | undefined = this.getGitInfo();
-    if (repoInfo && repoInfo.worktreeGitDir) {
-      return path.join(repoInfo.worktreeGitDir, 'hooks');
+    const hooksResult: IResultOrError<string> = this._tryGetGitHooksPath();
```

Comment:
You could probably handle the error in a method similar to how we handle the error in `getGitEmail()` here: https://github.com/microsoft/rushstack/blob/0f2d6a586a16a6970f7d6614d652b3bdd5b086bf/apps/rush-lib/src/logic/Git.ts#L100